src/main/java/ua.horobets/hw17/Order.java

```diff
@@ -0,0 +1,25 @@
+package ua.horobets.hw17;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+@Entity
+@Table(schema = "public",name = "order")
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@ToString(exclude = "users")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+    @Column(name = "orders",length = 15)
+    private int orders;
+    @Column(name = "name",length = 15)
+    private String name;
+    @ManyToOne(cascade = CascadeType.ALL)
+    @JoinColumn(name = "users")
```

Comment:
join column should be some id, since it's foreign key on some table.

So, good naming is user_id